<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetDoc</class>
 <widget class="QWidget" name="WidgetDoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="comboBoxComponents">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>File</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Group</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LOD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PAT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Switch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;&gt;&gt; Smoke</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;&gt;&gt; Rotor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;&gt;&gt; FLOLS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;&gt;&gt; Runway</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;&gt;&gt; World</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;&gt;&gt; Flame</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>Components</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButtonRemove">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
